cr/audit-service/oas/AuditServiceAPI.yaml

```
@@ -0,0 +1,218 @@
+openapi: 3.0.0
+info:
+  title: Audit Service API
+  description: This API allows posting audit events.
+  version: 1.0.0
+servers:
+  - url: http://localhost:9093
+info:
```

YAMLlint (1.38.0)

[error] duplication of key "info" in mapping

(key-duplicates)